src/introduction.md

```diff
@@ -9,38 +9,133 @@ provides three kinds of material:
+
+* Statements that define a term contain that term in *italics*. Whenever that
+  term is used outside of that chapter, it is usually a link to the section that
+  has this definition.
+
+  An *example term* is an example of a term beind defined.
+
+* Notes that contain useful information about the state of the book or point out
+  useful, but mostly out of scope, information are in blockquotes that start
+  with the word "Note:" in **bold**.
+
+  > **Note**: This is an example note.
+
+* Warnings that show unsound behavior in the language or possibly confusing
+  interactions of language features are in a special warning box.
+
+  <div class="warning">
+
+  Warning: This is an example warning.
+
+  </div>
+
+* Code snippets inline in the text are inside `<code>` tags.
+
+  Longer code examples are in a syntax highlighted box that has controls for
+  copying, executing, and showing hidden lines in the top right corner.
+
+  ```rust
+  # // This is a hidden line.
+  fn main() {
+      println!("This is a code example");
+  }
+  ```
+
+* The grammar and lexical structure is in blockquotes with either "Lexer" or
+  "Syntax" in <sup>**bold** superscript</sup> as the first line.
+
+  > **<sup>Syntax</sup>**\
+  > _ExampleGrammar_:\
+  > &nbsp;&nbsp; &nbsp;&nbsp; `~` [_Expression_]\
+  > &nbsp;&nbsp; | `box` [_Expression_]
 
-You may also be interested in the [grammar].
+## Contributing
 
-You can contribute to this document by opening an issue or sending a pull
-request to [the Rust Reference repository].
+We welcome contributions of all kinds.
 
-N. B. This document may be incomplete. Documenting everything might take a
-while. We have a [big issue] to track documentation for every Rust feature,
-so check that out if you can't find something here.
+You can contribute to this book by opening an issue or sending a pull
+request to [the Rust Reference repository]. If this book does not answer
+your question, and you think its answer is in scope of it, please do not
+hesitate to file an issue or ask about it in the Rust docs channels on IRC or
+discord. Knowing what people use this book for the most helps direct our
+attention to making those sections the best that they can be.
 
 [book]: ../book/index.html
-[standard]: ../std/index.html
+[standard library]: ../std/index.html
 [grammar]: ../grammar.html
 [the Rust Reference repository]: https://github.com/rust-lang-nursery/reference/
 [big issue]: https://github.com/rust-lang-nursery/reference/issues/9
-[Unstable Book]: https://doc.rust-lang.org/nightly/unstable-book/
\ No newline at end of file
+[Unstable Book]: https://doc.rust-lang.org/nightly/unstable-book/
+[_Expression_]: expressions.html
+[cargo book]: ../cargo/index.html
+[cargo reference]: ../cargo/reference/index.html
+[expressions chapter]: expressions.html
+[lifetime of temporaries]: expressions.html#temporary-lifetimes
+[linkage]: linkage.html
+[rustc book]: ../rustc/index.html
+[undocumented]: undocumeted.html
```

Comment:
It might be helpful to add a link to the [Rustonomicon](https://doc.rust-lang.org/nomicon/).